db/migrate/20151116114008_retire_links_column_from_links_sets.rb

```diff
@@ -0,0 +1,5 @@
+class RetireLinksColumnFromLinksSets < ActiveRecord::Migration
+  def change
+    rename_column :link_sets, :links, :legacy_links
+  end
+end
```

Comment:
Leaving this out to be picked up later would mean also removing the alias removed here:
https://github.com/alphagov/publishing-api/commit/d4b174dab8ab7369521c1f1f5c8091f800ccd1fa

I will also need to refer to `links` as `content_item_links`. I leaving this migration and the commit mentioned above in this PR would make everything easier and less confusing for the next person that will work on this.